pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[tool.poetry]
+name = "b2share-new-dev"
```

Comment:
Should this file be located somewhere other than root? I think this causes pip install -e . to crash if it is not moved / renamed

Reply:
Maybe document poetry use for future dependency whack-a-mole. 

Once we have the dependencies configured we should store them to requirements.txt and move the poetry files to a different folder